### PR TITLE
*: fix Turkish translation

### PR DESCRIPTION
**Pull request by yutyo**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**
PS: Merge this only if its OK for @seppacar , @unickorn and @mehmetmalli

### CODE REVIEW

File: pages.tr/common/alias.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
 # alias
 
-> Takma adlar/kısayollar oluşturur -- bir komut dizesi ile değiştirilen sözcükler.
+> Takma adlar/kısayollar (bir komut dizesi ile değiştirilen sözcükler) oluşturur.
```
Comment (unickorn) [inline pages.tr/common/alias.md:3]: Should this also not be in imperative form ("oluştur")?

Comment (yutyo) [reply pages.tr/common/alias.md:3]: Sure but its OK like that too as this is the description and not an instruction.

Comment (unickorn) [reply pages.tr/common/alias.md:3]: Ah, all good then, my bad!

**Review by unickorn — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Review by seppacar — APPROVED** [review]
LGTM, thanks @yutyo.

**Comment by mehmetmalli** [comment]
LGTM @yutyo 👍

**Review by mehmetmalli — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**